src/main/java/com/amazon/opendistroforelasticsearch/security/filter/OpenDistroSecurityFilter.java

```diff
@@ -303,7 +303,7 @@ public int order() {
             }
 
             if(threadContext.getTransient(OPENDISTRO_SECURITY_USER_AND_ROLES) == null) {
-                threadContext.putTransient(OPENDISTRO_SECURITY_USER_AND_ROLES, user.getUserRolesString());
+                threadContext.putTransient(OPENDISTRO_SECURITY_USER_AND_ROLES, user.getUserRolesTenantString());
```

Comment:
why we are not changing this to USER_ROLES_TENANT ? 
I know that it was failing UT but we should be able to fix that? and what backward compatibility you were mentioning in your previous comment?

Reply:
I could not find reference to OPENDISTRO_SECURITY_USER_AND_ROLES but UT was failing so kept the name.
backward compatibility was for the value (with and without tenant info is tested with common-utils parsing)

Reply:
It will be necessary to rename both the constant and it's key (`OPENDISTRO_SECURITY_CONFIG_PREFIX + "user_and_roles"`). The name of the constant is less important, renaming the key may introduce backward compatibility issue.

Reply:
key (PENDISTRO_SECURITY_CONFIG_PREFIX + "user_and_roles") cannot be changed as it is being used by Alerting plugin. Changing the name of the constant is causing UT failure.

Reply:
Agreed in meeting to change it to getUserInfoString, OPENDISTRO_SECURITY_USER_INFO_THREAD_CONTEXT and (OPENDISTRO_SECURITY_CONFIG_PREFIX + "user_info") to make name agnostic in the future.